pkg/sdn/plugin/node.go

```diff
@@ -94,8 +93,12 @@ func (node *OsdnNode) Start() error {
 		return fmt.Errorf("Failed to set up iptables: %v", err)
 	}
 
-	var networkChanged bool
-	networkChanged, err = node.SubnetStartNode(node.mtu)
+	networkChanged, err := node.SetupSDN()
+	if err != nil {
+		return err
+	}
+
+	err = node.SubnetStartNode()
 	if err != nil {
 		return err
 	}
```

Comment:
I don't think retrying is likely to help; if we can't update the pod networking, then something is just broken. (Eg, OVS isn't running.) But I think if things are broken enough that UpdatePod() fails, then startup is going to fail for other reasons anyway